htmlbasic/menustyle.css

```diff
@@ -0,0 +1,18 @@
+.row {
```

Comment:
This file is never used in the 4 htmls and you can see that your file is also not responsive. It is called menustyle.css and in the imports, which are also not in the correct place (see remarks there), it is called menu.css